src/main/java/org/folio/entlinks/domain/entity/AuthorityArchive.java

```diff
@@ -0,0 +1,111 @@
+package org.folio.entlinks.domain.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.PostLoad;
+import jakarta.persistence.PrePersist;
+import jakarta.persistence.Table;
+import jakarta.persistence.Transient;
+import jakarta.persistence.Version;
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import lombok.With;
+import org.folio.entlinks.domain.entity.base.Identifiable;
+import org.hibernate.Hibernate;
+import org.hibernate.annotations.JdbcTypeCode;
+import org.hibernate.type.SqlTypes;
+import org.springframework.data.domain.Persistable;
+
+@Getter
+@Setter
+@With
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString
+@Table(name = "authority_archive")
+public class AuthorityArchive extends MetadataEntity implements Persistable<UUID>, Identifiable<UUID> {
```

Comment:
Can this entity be extended from the Authority entity to omit code duplication?

Reply:
Yes, it can be, using ```@MappedSuperclass```